src/test/java/org/isf/lab/test/TestLaboratory.java

```diff
@@ -91,7 +92,7 @@ public void check(Laboratory laboratory) {
 			assertThat(laboratory.getPatName()).isEqualTo(patName);
 		}
 		assertThat(laboratory.getDate()).isEqualTo(registrationDate);
-		assertThat(laboratory.getExamDate()).isEqualTo(examDate);
+		assertThat(laboratory.getDate()).isCloseTo(registrationDate, within(1, ChronoUnit.SECONDS));
```

Comment:
Why this isCloseTo comparing?

Reply:
I assumed it was handling rounding issues; it was what I found in the original attempt to use Java 8 time methods.

Reply:
Yes, maybe it depends by the underlying DBMS versions... 

Anyway the assertion at line 93 is on the same field :)

Moreover, the fields are defined as follow:

```private LocalDateTime registrationDate = LocalDateTime.now().minusWeeks(5);```
```private LocalDate examDate = LocalDate.now();```

So maybe the "within 1 second" check make sense on the registrationDate rather then examDate.

Reply:
This was a mistake; I've fixed it with the latest commit.